src/node-ram-cache.cpp

```diff
@@ -240,14 +240,6 @@ void node_ram_cache::set_dense(osmid_t id, osmium::Location location)
         }
 
         if (queue[expectedpos] != &blocks[block]) {
-            if (m_warn_node_order) {
-                log_warn("Found out of order node {}"
-                         " ({},{}) - this will impact the cache efficiency",
-                         id, block, offset);
-
-                // Only warn once
-                m_warn_node_order = false;
-            }
```

Comment:
This leaves us with an odd 'if' check that is not quite understandable. If you don't want to touch the code any further than removing the check, I would suggest to add a comment about what used to be here aka 'This can only happen if the node is out of order. Ignore the node.'

Reply:
I added a comment.